main.py

```diff
@@ -2,7 +2,6 @@
 import sys
 from base_util import LOG_FORMAT
 from config import input_uri, output_uri
```

Comment:
This was from prior code when we were thinking of running this worker as a job (which sadly isn't possible). I could remove it and have the worker be exclusively a service at this point since the job isn't used for anything other than testing (which even could be tested via the service way).